src/screens/discover/components/ENSCreateProfileCard.tsx

```diff
@@ -1,10 +1,10 @@
 import lang from 'i18n-js';
 import React, { useCallback, useEffect, useMemo } from 'react';
-import RainbowGrid from '../../assets/discover-profiles-card.png';
-import { useNavigation } from '../../navigation/Navigation';
-import { ButtonPressAnimation } from '../animations';
-import { ensAvatarUrl } from '../ens-registration/IntroMarquee/IntroMarquee';
-import ImgixImage from '../images/ImgixImage';
+import RainbowGrid from '@/assets/discover-profiles-card.png';
+import { useNavigation } from '@rainbow-me/navigation/Navigation';
```

Comment:
tsk tsk

Reply:
Will change the import, thanks for spotting that